src/services/AuthService.tsx

```diff
@@ -194,7 +194,13 @@ const Auth = ({ children }: GenericContextProviderProps) => {
 				);
 		};
 
-		if (!initialized || !keycloak.authenticated) return;
+		if (!initialized || !keycloak.authenticated) {
```

Comment:
Here is a table documenting what happens in the code:

| initialized | keycloak.authenticated | !initialized \|\| !keycloak.authenticated | initialized && !keycloak.authenticated | logout() | return |
|-------------|------------------------|-------------------------------------------|----------------------------------------|----------|--------|
| false       | false                   | true                                      | false                                  | no       | yes    |
| false       | true                    | true                                      | false                                  | no       | yes    |
| true        | false                   | true                                      | true                                   | yes      | yes    |
| true        | true                    | false                                     | false                                  | no       | no     |